// containers/item/index.js
import React, {Component} from 'react'
import {connect} from 'react-redux'
import {Link} from 'react-router-dom'

import {fetchItemById, addItemToBasket} from 'actions'
import {getItemById} from 'selectors'
import Navbar from 'components/navbar'
import styles from './item.module.css'

class Item extends Component {
  componentDidMount() {
    const itemId = this.props.match.params.id
    this.props.fetchItemById(itemId)
  }

  renderField() {
    const {item} = this.props

    return (
      <div className='column'>
        <div className='ab-details-title'>
          title
        </div>
        <div className='ab-details-info'>
         info
        </div>
      </div>
    )
  }

  renderContent() {
    const {item} = this.props

    return (
            <img
              className={`img-fluid ${styles.fluid}`}
              src={item.url}
              alt={item.name}
            />
    )
  }

  renderSidebar() {
    const {item, addItemToBasket} = this.props

    return (
      <div className='col-xl-7 col-lg-7 col-md-6 col-sm-12 col-12'>
        <div className={`d-flex justify-content-sm-center flex-column ${styles.sidebar}`}>
          <span className='h1'>{item.name}</span>
          <span className='h2'>₴ {item.price}</span>
        </div>

        <button
          type='button'
          className={`btn btn-block ${styles.btn}`}
          onClick={() => addItemToBasket(item.id)}
        >
          Add to cart
        </button>
      </div>
    )
  }

  render() {
    return (
      <div className='view-container'>
        <Navbar/>
        <div className='container'>
          <div className='row'>
            <div className='col-12 col-sm-12 col-md-6 d-flex justify-content-md-end justify-content-lg-end justify-content-xl-end justify-content-center'>
              {this.props.item && this.renderContent()}
            </div>
            <div className='col-12 col-sm-12 col-md-6 d-flex justify-content-md-center'>
              {this.props.item && this.renderSidebar()}
            </div>
          </div>
        </div>
      </div>
    )
  }
}

const mapStateToProps = state => {
  return {
    item: getItemById(state, state.itemPage.id)
  }
}

const mapDispatchToProps = {
  fetchItemById,
  addItemToBasket
}

export default connect(mapStateToProps, mapDispatchToProps)(Item)
