// api/mockClothes.js
export default [
  {
    id: "1",
    categoryId: "1",
    name: "FREEDOM.",
    description: "",
    price: "600",
    url: "/uploads/Freedom.jpg",
    sizes: "S, M, L, XL",
    dimensions: "30.0 x 20.0 x 10.0 см",
    color: "Black, Yellow"
  },
  {
    id: "2",
    categoryId: "1",
    name: "BALLOON.",
    description: "",
    price: "850",
    url: "/uploads/Balloon.jpg",
    sizes: "S, M, L",
    dimensions: "30.0 x 20.0 x 10.0 см",
    color: "Black, Yellow"
  },
  {
    id: "3",
    categoryId: "1",
    name: "CHAOS.",
    description: "",
    price: "700",
    url: "/uploads/Chaos.jpg",
    sizes: "M, L, XL",
    dimensions: "30.0 x 20.0 x 10.0 см",
    color: "Black, Yellow"
  },
  {
    id: "4",
    categoryId: "1",
    name: "DISAPPEARED.",
    description: "",
    price: "1000",
    url: "/uploads/Disappeared.jpg",
    sizes: "S, M, L",
    dimensions: "30.0 x 20.0 x 10.0 см",
    color: "Black, Yellow"
  },
  {
    id: "5",
    categoryId: "1",
    name: "GANGSTA.",
    description: "",
    price: "950",
    url: "/uploads/Gangsta.jpg",
    sizes: "S, M, L",
    dimensions: "30.0 x 20.0 x 10.0 см",
    color: "Black, Yellow"
  },
  {
    id: "6",
    categoryId: "1",
    name: "SAFE.",
    description: "",
    price: "550",
    url: "/uploads/Safe.jpg",
    sizes: "L, XL",
    dimensions: "30.0 x 20.0 x 10.0 см",
    color: "Black, Yellow"
  },
  {
    id: "7",
    categoryId: "1",
    name: "SAVAGE.",
    description: "",
    price: "600",
    url: "/uploads/Savage.jpg",
    sizes: "S, M, L, XL",
    dimensions: "30.0 x 20.0 x 10.0 см",
    color: "Black, Yellow"
  },
  {
    id: "8",
    categoryId: "1",
    name: "SKELPRAY.",
    description: "",
    price: "900",
    url: "/uploads/Skelpray.jpg",
    sizes: "S, M, L, XL",
    dimensions: "30.0 x 20.0 x 10.0 см",
    color: "Black, Yellow"
  },
  {
    id: "9",
    categoryId: "1",
    name: "SMILEY.",
    description: "",
    price: "650",
    url: "/uploads/Smiley.jpg",
    sizes: "S, M",
    dimensions: "30.0 x 20.0 x 10.0 см",
    color: "Black, Yellow"
  },
  {
    id: "10",
    categoryId: "1",
    name: "SPACEROCKET.",
    description: "",
    price: "1000",
    url: "/uploads/SpaceRocket.jpg",
    sizes: "S, M, L, XL",
    dimensions: "30.0 x 20.0 x 10.0 см",
    color: "Black, Yellow"
  }
]
